Makefile

```diff
@@ -748,56 +747,56 @@ venv/bin/protoc-gen-grpc-web: $(var.)GRPC_WEB_VERSION $(var.)GRPC_WEB_PLATFORM |
 	curl -o $@ -L --fail https://github.com/grpc/grpc-web/releases/download/$(GRPC_WEB_VERSION)/protoc-gen-grpc-web-$(GRPC_WEB_VERSION)-$(GRPC_WEB_PLATFORM)
 	chmod 755 $@
 
-pkg/apis/kat/echo.pb.go: kat/api/echo.proto venv/bin/protoc venv/bin/protoc-gen-gogofast
+pkg/apis/kat/echo.pb.go: ambassador/api/kat/echo.proto venv/bin/protoc venv/bin/protoc-gen-gogofast
 	./venv/bin/protoc \
-		--proto_path=$(CURDIR)/kat/api \
+		--proto_path=$(CURDIR)/ambassador/api/kat \
 		--plugin=$(CURDIR)/venv/bin/protoc-gen-gogofast --gogofast_out=plugins=grpc:$(@D) \
 		$(CURDIR)/$<
 
-kat/sandbox/grpc_web/echo_grpc_web_pb.js: kat/api/echo.proto venv/bin/protoc venv/bin/protoc-gen-grpc-web
+tools/sandbox/grpc_web/echo_grpc_web_pb.js: ambassador/api/kat/echo.proto venv/bin/protoc venv/bin/protoc-gen-grpc-web
```

Comment:
Hm. Do we really use the sandbox any more? I just realized I don't actually know what it's really for. :wink:

Reply:
I don't know. I can remove it if it is not neccessary.

Reply:
It's from kat-backend.git, so I guess it's useful for an inner loop when hacking on kat-client or kat-server?